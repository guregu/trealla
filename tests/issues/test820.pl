:- initialization(main).

main :-
	 X = "abc"||(1*(1+2)),
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "abc"||-123,
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "abc"||_,
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "abc"||"def",
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "abc"||f(_),
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "abc"||[1,2,3],
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "abc"||{1,2,3},
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "ab'c"||[],
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "ab`c"||[],
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "ab\`c"||_,
	 write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "ab\'c"||_,
	  write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "ab,c"||_,
	  write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "ab|c"||_,
	  write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "ab]c"||_,
	  write_term(X,[double_quotes(true)]), nl, fail.
main :-
	 X = "ab[c"||_,
	  write_term(X,[double_quotes(true)]), nl, fail.
main.
