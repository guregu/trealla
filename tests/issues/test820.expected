"abc"||(1*(1+2))
"abc"||-123
"abc"||_0
"abcdef"
"abc"||f(main)
[a,b,c,1,2,3]
"abc"||{1,2,3}
"ab'c"
"ab\`c"
"ab\`c"||_0
"ab'c"||_0
"ab,c"||_0
"ab|c"||_0
"ab]c"||_0
"ab[c"||_0
